### PR TITLE
fix: more conditional structures python

### LINKED ISSUE #38926: Python for Everybody - More Conditional Structures
**Describe your problem and how to reproduce it:**
The answer to this is incorrect. Due to the existance of the `try...catch`, the code does not get interrupted, and runs to completion:
```
temp = "5 degrees"
cel = 0
try:
	fahr = float(temp)
	cel = (fahr - 32.0) * 5.0 / 9.0
except:
	print("temp should be a number")
print(cel)
```

**Add a Link to the page with the problem:**
https://www.freecodecamp.dev/learn/scientific-computing-with-python/python-for-everybody/more-conditional-structures

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

### PR DESCRIPTION
**Pull request by niteshseram**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #38926

Made some changes to the question as well answer.

### CODE REVIEW

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/more-conditional-structures.english.md [hunk 1]
```diff
@@ -8,43 +8,42 @@ videoId: HdL82tAZR20
...
-        6
+      3,4
     - |
-        7
+      4
     - |
-        None
-  solution: 2
-```
+      None
```
Comment (Sky020) [inline curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/more-conditional-structures.english.md:45]:
```suggestion
        None
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Just in keeping with the rest of the MD format.

**Review by ShaunSHamilton — DISMISSED** [review]
LGTM 👍

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]

**Comment by niteshseram** [comment]
@scissorsneedfoodtoo I have made the changes. Have a look if it is fine.

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/more-conditional-structures.english.md [hunk 2]
```diff
@@ -8,43 +8,42 @@ videoId: HdL82tAZR20
...
-        fahr = float(temp)
-        cel = (fahr - 32.0) * 5.0 / 9.0
-    except:
-        print("temp should be a number")
+    fahr = float(temp)
+    cel = (fahr - 32.0) * 5.0 / 9.0
     print(cel)
     ```
-    Which line would cause the script to immediately stop because of an error?
+    Which line/lines should be surrounded by a try and catch block?
```
[the remark this answers is not in the document]
Comment (niteshseram) [reply curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/more-conditional-structures.english.md]: @scissorsneedfoodtoo I have made the changes. Have a look if it is fine.

[the remark this answers is not in the document]
Comment (Sky020) [reply curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/more-conditional-structures.english.md]: @scissorsneedfoodtoo , thank you, for catching that. I have spent too much time with JS.

**Review by moT01 — COMMENTED** [review]
I see that there is an [open PR](https://github.com/freeCodeCamp/freeCodeCamp/pull/39005) to add line numbers to this challenge, do we want to add those here?

Edit: Also, it seems like the code doesn't execute if there's no `except` block...
Screen Shot 2020-06-04 at 10 24 14 AM
With except block, all the code executes... (output is on the right)
Screen Shot 2020-06-04 at 10 24 37 AM
Not sure if we want to adjust this further or not. You could maybe put an `except` block in there to start.
```
temp = "5 degrees"
cel = 0
fahr = float(temp)
cel = (fahr - 32.0) * 5.0 / 9.0
except:
  print('error converting to celsius')
print(cel)
```

I think you could also possibly argue that the solution could be answer 2 (line 3), 3 (lines 3 and 4), or 4 (line 4). The try block could go anywhere you want really. Perhaps there should be more clarification in the question. `Which line(s) should be in a 'try' block to check if 'temp' got converted to a float and 'cel' was properly calculated?` Although, that seems to make the answer too obvious.

How about `Which lines(s) should be in a 'try' block to ensure all the code is executed?` And the example looks like my above example.

**Comment by niteshseram** [comment]
> How about `Which lines(s) should be in a 'try' block to ensure all the code is executed?` And the example looks like my above example.

@moT01 I think this looks good to me. And about line numbers, even I was thinking to add. Should I go for it here itself?

**Comment by moT01** [comment]
I don't mind you going for it, I think it will be accepted - but someone may come in with a minor tweak or alternate suggestion.

File: curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/more-conditional-structures.english.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (scissorsneedfoodtoo) [reply curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody/more-conditional-structures.english.md]: @niteshseram, great, thanks for making those changes. And no problem @Sky020, I barely noticed it myself for the same reason!

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Comment by scissorsneedfoodtoo** [comment]
The changes @moT01 suggested sound great, and should cover all issues where another answer is possible, or the question is seen as invalid because it wouldn't run without adding `except`.

**Pull request merged**